standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -9826,7 +9826,7 @@ public FireEventResponse fire_listener_event(FireEventRequest rqst) throws TExce
       catName = rqst.isSetCatName() ? rqst.getCatName() : getDefaultCatalog(conf);
       dbName = rqst.getDbName();
       tblName = rqst.getTableName();
-      List<String> partitionVals = rqst.getPartitionVals();
```

Comment:
should we also consider the old client in case `rqst.getPartitionVals()` is not null?

Reply:
Ack